recipes/rebound/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "REBOUND" %}
+{% set version = "3.4.0" %}
+{% set sha256 = "d070bfd7a8c48a28776b4bae5ba4dcf22d81e7776cd518dc3a3479972104d5bc" %}
+
+package:
+  name: rebound
+  version: "3.4.0"

```

Comment:
Use variables here for name and version instead of numbers -- see see this recipe for an example: https://github.com/conda-forge/staged-recipes/pull/3011/files